.ci/azure-pipelines.yml

```diff
@@ -0,0 +1,98 @@
+            echo "##vso[task.setvariable variable=CERTIFICATE_DIR]$(Agent.TempDirectory)/certificates"
+          failOnStderr: false
+
+      - task: UseNode@1
+        displayName: Install Node
+        inputs:
+          version: '12.x'
+
+      - task: CmdLine@2
+        displayName: Install Jellyfin Tizen
+        inputs:
+          script: | 
+            yarn install
+            JELLYFIN_WEB_DIR=./node_modules/jellyfin-web/dist yarn build
+          failOnStderr: false
+
+      - task: CmdLine@2
+        displayName: Setup Tizen Studio
+        inputs:
+          script: | 
+            export TIZEN_CLI_INSTALLER_URL=http://download.tizen.org/sdk/Installer/tizen-studio_3.7/web-cli_Tizen_Studio_3.7_ubuntu-64.bin
+            export TIZEN_STUDIO=$TIZEN_HOME_DIR/tizen-studio
+            
+            mkdir -p $TIZEN_HOME_DIR
+            cd $TIZEN_HOME_DIR
+
+            INSTALLER=$(Agent.TempDirectory)/$(basename "$TIZEN_CLI_INSTALLER_URL") \
+              && curl "$TIZEN_CLI_INSTALLER_URL" > "$INSTALLER" \
+              && chmod 755 "$INSTALLER" \
+              && "$INSTALLER" --no-java-check --accept-license $TIZEN_STUDIO \
+              && rm "$INSTALLER" \
+              && cd $TIZEN_STUDIO/tools/certificate-generator/patches/partner \
+              && ./patch.sh $TIZEN_STUDIO \
+              && rm -rf \
+                $TIZEN_HOME_DIR/.package-manager \
+                $TIZEN_HOME_DIR/tizen-studio/license \
+                $TIZEN_HOME_DIR/tizen-studio/package-manager            
+          failOnStderr: false
+  
+      - task: CmdLine@2
+        displayName: Setup certificates
+        inputs:
+          script: |
+            export PATH=$TIZEN_HOME_DIR/tizen-studio/tools:$TIZEN_HOME_DIR/tizen-studio/tools/ide/bin:$PATH
+
+            mkdir -p $CERTIFICATE_DIR
+            
+            echo $(CERTIFICATE) | base64 --decode > $CERTIFICATE_DIR/certificate.p12
+            tizen security-profiles add -n DebugProfile -a $CERTIFICATE_DIR/certificate.p12 -p $CERTIFICATE_PASSWORD
+
+      - task: CmdLine@2
+        displayName: Package build
+        inputs:
+          script: |
+            export PATH=$TIZEN_HOME_DIR/tizen-studio/tools:$TIZEN_HOME_DIR/tizen-studio/tools/ide/bin:$PATH
+            tizen build-web -e ".*" -e gulpfile.js -e README.md -e "node_modules/*" -e "jellyfin-web/*" -e "package*.json" -e "yarn.lock"
+            tizen package -t wgt -s debug -- .buildResult
+
+      - task: CopyFiles@2
+        displayName: Staging artifacts
+        inputs:
+          SourceFolder: '.buildResult'
+          Contents: '*.wgt'
+          TargetFolder: '$(Build.ArtifactStagingDirectory)'
+          OverWrite: true
+
+      - task: GitHubRelease@1
+        displayName: Creating Github Release
+        inputs:
+          gitHubConnection: 'github.com_robinkruyt'
+          repositoryName: '$(Build.Repository.Name)'
+          action: 'create'
```

Comment:
We actually update versions by adding a release on GitHub so edit would match the current behavior, although it might make sense to change this.

Reply:
I'm not super familiar with Azure Pipelines. How would this work in practice? If I change the action to Edit, I assume we need to somehow match the build from the pipeline with a particular (pre-)release. How is this done?

Reply:
We would draft a release on GitHub, which would automatically create a tag as well. The pipeline would run and then during the publish step it would modify the existing release to append the binaries with the `edit` action.